### PR TITLE
fix: send token example

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

File: pages/dev/send-tokens/overview.mdx [hunk 1]
```diff
@@ -44,7 +44,9 @@ function approve(address spender, uint256 amount) external returns (bool);
 
 Here, `spender` is the Gateway address on the source chain.
 
-Find a list of assets, their names and their addresses: [Mainnet](../reference/mainnet-contract-addresses) | [Testnet](../reference/testnet-contract-addresses).
+Find a list of assets, their names and their addresses: [Mainnet](../reference/mainnet-contract-addresses) | [Testnet](../reference/testnet-contract-addresses). Note that for the same asset denom,
```
Comment (StephenFluin) [inline pages/dev/send-tokens/overview.mdx:47]: all of this content should be on one line

File: pages/dev/send-tokens/overview.mdx [hunk 2]
```diff
@@ -44,7 +44,9 @@ function approve(address spender, uint256 amount) external returns (bool);
...
+the ERC20 symbol could be different on different chains.
+For e.g., `USDC` on Ethereum is linked to `axlUSDC` on Avalanche.
```
Comment (StephenFluin) [inline pages/dev/send-tokens/overview.mdx:49]: The correct english format  for eg is to use this as a parenthetical statement attached to the previous statement.
"the ERC20 symbol could be different on different chains (e.g. USDC on ethereum..."
or you could just use words in your separate sentence.
"For example, `USDC` on ethereum is linked..."

File: pages/dev/send-tokens/overview.mdx [hunk 3]
```diff
@@ -54,7 +56,7 @@ Call `sendToken` on the Gateway contract of the source chain. Example:
 sendToken(
     "avalanche", // destination chain name
     "0xF16DfB26e1FEc993E085092563ECFAEaDa7eD7fD", // some destination wallet address (should be your own)
-    "USDC", // asset symbol
+    "axlUSDC", // asset symbol, can be differ by chain, see above.
```
Comment (StephenFluin) [inline pages/dev/send-tokens/overview.mdx:59]: remove period at the end

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: pages/dev/send-tokens/overview.mdx (hunk 2 above)
Comment (milapsheth) [inline pages/dev/send-tokens/overview.mdx:49]:
```suggestion
Find a list of assets, their names and their addresses: [Mainnet](../reference/mainnet-contract-addresses) | [Testnet](../reference/testnet-contract-addresses). Note that for the same asset denom, the ERC20 symbol could be different on different chains (e.g. `USDC` on Ethereum is linked to `axlUSDC` on Avalanche).
```

File: pages/dev/send-tokens/overview.mdx (hunk 3 above)
Comment (milapsheth) [inline pages/dev/send-tokens/overview.mdx:59]:
```suggestion
    "axlUSDC", // asset symbol, can be differ by chain, see above
```

**Pull request merged**